app/routes.py

```diff
@@ -63,3 +64,37 @@ def post_kml():
         ),
         201
     )
+
+
+@app.route('/kml/<kml_admin_id>', methods=['GET'])
+def get_id(kml_admin_id):
+    logger.debug("entering route /kml/%s", kml_admin_id)
+    enforcer = DynamoDBFilesHandler(
+        table_name=AWS_DB_TABLE_NAME,
+        bucket_name=AWS_S3_BUCKET_NAME,
+        table_region=AWS_DB_REGION_NAME,
+        endpoint_url=AWS_DB_ENDPOINT_URL
+    )
+    item = enforcer.get_item(kml_admin_id)
+
+    logger.debug(item)
+    # Fetching a non existing Item will return "None"
+    if item is None:
+        logger.info("exiting /kml/%s with a 400 status", kml_admin_id)
+        abort(400, f"{kml_admin_id} is not an existing kml id.")
```

Comment:
I'm not sure if the logger.debugg is here needed or the errorhandler in _init_.py could also handle that.

Reply:
the logger.debug was indeed not necessary.